cpp/modmesh/view/RPythonConsoleDockWidget.cpp

```diff
@@ -129,10 +129,19 @@ void RPythonConsoleDockWidget::executeCommand()
     m_command_string = "";
     m_current_command_index = static_cast<int>(m_past_command_strings.size());
     auto & interp = modmesh::python::Interpreter::instance();
-    modmesh::python::PyStdErrOutStreamRedirect pyOutputRedirect{};
+
+#ifdef DISABLE_REDIRECT
```

Comment:
I think we should not use a compile-time macro for this simple switch.  The switch controls something outside a loop (not to say a tight loop), so we do not worry about its runtime.  A runtime switch would do.

Reply:
The runtime switch can be added on `RPythonConsoleDockWidget` and exposed to Python, so that we can control it from the console itself.